content/pt-br/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Segurança para Nós Windows
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página descreve considerações de segurança e boas práticas específicas para o sistema operacional Windows.
+
+<!-- body -->
+
+## Proteção para dados Secret nos Nós
+
+No Windows, os dados do Secret são escritos em texto não-encriptado no Nó local do
+armazenamento (em comparação ao uso de tmpfs / sistemas de arquivo em memória no Linux). Como um operador do cluster, você deve tomar as duas medidas adicionais a seguir:
+
+1. Use ACLs em arquivos para proteger a localização do arquivo Secrets.
+2. Aplicar criptografia a nível de volume usando
+   [BitLocker](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-how-to-deploy-on-windows-server).
+
+## Usuários dos Contêineres
+
+[RunAsUsername](/docs/tasks/configure-pod-container/configure-runasusername)
+pode ser utilizado para Pods com Windows ou contêiner para executar os processos do contêiner como usuário específico. Isto é aproximadamente equivalente a
```

Comment:
"Pods com Windows ou contêiner" is not quite right. I suggest "Pods ou contêineres com Windows" for this specific sentence.

Reply:
Done!